kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -14333,6 +14333,13 @@ public String expandMacros(String url) {
         try {
             ContentResource cr = getResource(ref);
 
+            long contentLength = cr.getContentLength();
+
+            if (contentLength > (10 * 1024 * 1024)) {
```

Comment:
I think hardcoding 10 is not a good idea.

Reply:
Why not? This error basically brings down your Sakai instance. It could be a Sakai property, but who in their right minds wants to do trial and error with it?

Reply:
If always the Sakai instance brings down with more than 10, I'll put numbers (10 and 1024) as String final, which is better than 10 * 1024 *1024.
I don't have access to this issue.

Reply:
I've made it a property, but still with a default of 10MB. We saw a file of over 150MB being uploaded and bringing a client's node down, which is obviously a crazy size, but most of my docx files hardly every reach 2MB.